rose/client/main.py

```diff
@@ -74,6 +74,14 @@ def main():
         d = {}
         exec f in d, d
 
-    reactor.connectTCP(d['server_address'], config.game_port,
+    if len(sys.argv) == 3:
+        server_address = sys.argv[2]
+    else:
+        server_address = d['server_address']
+
+    print ('Client \"%s\" connecting to Server \"%s\"'
```

Comment:
Escaping quotes is not needed when using single quotes. Also, if you want to quote the arguments (good idea when the value provided by the user, you should use %r - which show the value quoted.

print is not a function in python 2. Using print() works since it ("string) is a string, but it confuse the reader. We don't support python 3 yet and converting to python 3 requires importing print_function from _future__, you can see how this is done in vdsm.

I would also change the message to:

   print "Client %r connecting to %r" % (server_address, d["driver_name"])\

Or:
    print "Connecting to %r as %r" %  (server_address, d["driver_name"])